java/ingest/ingest-runner/src/main/java/sleeper/ingest/impl/IngesterIntoPartitions.java

```diff
@@ -111,7 +115,17 @@ private static List<CompletableFuture<FileReference>> closeMultiplePartitionFile
      */
     public CompletableFuture<List<FileReference>> initiateIngest(
             CloseableIterator<Record> orderedRecordIterator, PartitionTree partitionTree) throws IOException {
+        if (ingestFileWritingStrategy == IngestFileWritingStrategy.ONE_FILE_PER_LEAF) {
+            return ingestOneFilePerLeafPartition(orderedRecordIterator, partitionTree);
+        } else if (ingestFileWritingStrategy == IngestFileWritingStrategy.ONE_REFERENCE_PER_LEAF) {
+            return ingestOneFileWithReferencesInLeafPartitions(orderedRecordIterator, partitionTree);
+        } else {
+            throw new IllegalArgumentException("Unknown ingest mode: " + ingestFileWritingStrategy);
```

Comment:
This message could refer to it as a file writing strategy rather than a mode.

Reply:
Done, thanks